### DISCUSSION TITLE
slurm-web-restapi and slurm-web-dashboard packages not found

### DISCUSSION THREAD

**Discussion by balajin2022** [rackslab/Slurm-web#240] [Q&A] [answered]
i have tried to install slurm-web-restapi and slurm-web-dashboard packages from github repository, docker hub by searching the images but i did not find these packages anywhere, could any body has solution of it.

**Comment by rezib** [comment, marked as answer]
Hello @balajin2022, I converted the issue into a discussion as this is not a factual bug but a consequence of the current situation.

Packages for versions 2.x are not distributed, you must build them by yourself. Packaging code for `*.deb` packages (Debian or Ubuntu) is available in [`debian/`](https://github.com/rackslab/slurm-web/tree/2.x/debian) folder. Packaging code for `*.rpm` packages (RHEL and consorts) is available in [`rhel/`](https://github.com/rackslab/slurm-web/tree/2.x/rhel) folder.

Starting with upcoming version 3, as mentioned in https://github.com/rackslab/slurm-web/discussions/235, [Rackslab](https://github.com/rackslab) will distribute prebuilt RPM and deb packages (for Slurm-web and its dependencies) ready to use for major Linux distributions, along with Docker images. It still requires some work though.

**Reply by balajin2022** [reply]
Thank you for your reply, one thing i would like to know, have you defined any process to build them by myself so will follow that , actually i am using Ubuntu 22.04, please help me in this case, that would be appreciated

**Comment by rezib** [comment]
This question was about deprecated Slurm-web 2, then I close the discussion.
